routes/permafrost.py

```diff
@@ -236,4 +253,63 @@ def run_point_fetch_all_permafrost(lat, lon):
         "obupfx": package_obu_vector(gs_results[2]),
     }
 
+    csv_dicts = []
+    if request.args.get("format") == "csv":
+        data = nullify_and_prune(data, "permafrost")
+        if data in [{}, None, 0]:
+            return render_template("404/no_data.html"), 404
+
+        fieldnames = [
+            "source",
+            "era",
+            "model",
+            "scenario",
+            "variable",
+            "value",
+        ]
+
+        gipl_data = {"gipl": data["gipl"]}
+        csv_dicts += build_csv_dicts(
+            gipl_data,
+            fieldnames[0:-1],
+        )
+
+        # Non-GIPL values have a simpler nesting structure and need to be
+        # handled separately.
+        non_gipl_fields = [
+            "source",
+            "variable",
+            "value",
+        ]
+        for source in ["jorg", "obu_magt", "obupfx"]:
+            subset = {source: data[source]}
+            csv_dicts += build_csv_dicts(
+                subset,
+                non_gipl_fields,
+            )
+
+        place_id = request.args.get("community")
+        place_name, place_type = place_name_and_type(place_id)
+
+        metadata = csv_metadata(place_name, place_id, place_type, lat, lon)
+        metadata += "# alt is the active layer thickness in meters\n"
+        metadata += "# magt is the mean annual ground temperature in degrees Celsius\n"
+        metadata += "# ice is the estimated ground ice volume\n"
+        metadata += "# pfx is the permafrost extent\n"
+        metadata += "# 2025 represents 2011 - 2040\n"
+        metadata += "# 2050 represents 2036 - 2065\n"
+        metadata += "# 2075 represents 2061 – 2090\n"
```

Comment:
Need to be changed to regular dashes just to match, but I made another ticket for that.